modules/trusted-firmware-m/Kconfig.tfm

```diff
@@ -177,6 +177,28 @@ config TFM_PARTITION_PLATFORM_CUSTOM_REBOOT
 	  Instead the application will have to override the weak ARM
 	  implementation of sys_arch_reset().
 
+config TFM_DUMMY_PROVISIONING
+	bool "Provision with dummy values. NOT to be used in production"
+	select TFM_INITIAL_ATTESTATION_KEY
+	default n if BOARD_B_U585I_IOT02A
```

Comment:
I think I should move this to a defconfig instead.

Reply:
Did you forget to delete this line?

I see it is in a defconfig.

Reply:
No, this is a self-review comment / reminder.

The defconfig is a different symbol, but the defaults are related.

Reply:
Updated.